src/druid628/exactTarget/EtClient.php

```diff
@@ -28,7 +28,7 @@ class EtClient extends EtBaseClass
     const SOAPWSDL = "http://exacttarget.com/wsdl/partnerAPI";
 
     /** @var \druid628\exactTarget\EtSoapClient $client */
-    protected $client;
+    public $client;
```

Comment:
We're using it internally to get the soap request and response for debugging in certain environments, although it can be changed back